### PR TITLE
[DOC] Reporting documentation

### PR DESCRIPTION
**Pull request by javiergcas**
Closes None.

Changes proposed in this pull request:

- First draft of documentation for reports
- Figures described
- Interaction description missing

### CODE REVIEW

**Comment by eurunuela** [comment]
I guess we're waiting on the new reports to be merged, before we merge this PR.

**Comment by eurunuela** [comment]
We haven't forgotten about this stale bot!

File: docs/reporting.rst
Comment (smoia) [inline docs/reporting.rst]:
```suggestion
  about classification status. In this plot, size is not related to variance explained.
```

File: docs/reporting.rst
Comment (smoia) [inline docs/reporting.rst]:
```suggestion
  about classification status. In this plot, size is not related to variance explained.
```

File: docs/reporting.rst
Comment (smoia) [inline docs/reporting.rst]:
```suggestion
  by each individual component, as well as the total variance explained by each of the three
```

File: docs/reporting.rst
Comment (smoia) [inline docs/reporting.rst]: Is this a question for the reviewer or something that had to be checked but was not?

Comment (eurunuela) [reply docs/reporting.rst:78]: ? and . are close to each other in the American keyboard. So, it's probably just a typo.

**Comment by eurunuela** [comment]
I can commit those final changes @smoia, then we're good to go once the reports are merged.

**Comment by javiergcas** [comment]
@smoia changes have been accepted. Please review it.

**Comment by jbteves** [comment]
@javiergcas Thanks for pushing those changes! For future reference on the GitHub UI, you can go under "Reviewers" (highlighted with red box) and click on the refresh symbol next to the reviewer's username (shown with bottom arrow), and then that user should get a review-specific notification. You can also request additional reviewers by clicking on the gear icon (top arrow) and searching for a username.

**Comment by emdupre** [comment]
I think we're ready to work on these now that the reporting functionality is merged ! :tada: 

@smoia can I ask you to re-review and see if your changes were addressed ?

**Review by eurunuela — DISMISSED** [review]
LGTM! Would it be a good idea to add the link to the OHBM demo reports too? @emdupre @jbteves @handwerkerd @tsalo  @dowdlelt

File: docs/reporting.rst
Comment (tsalo) [inline docs/reporting.rst:65]: These plots do not currently appear to be interactive. Also, the brain plot is not described here.

File: docs/reporting.rst
Comment (tsalo) [inline docs/reporting.rst:156]:
```suggestion
  Specific user interactions can be switched on/off by clicking on their associated icon within
```

**Comment by tsalo** [comment]
I have a couple of minor requested changes, but we also need for this new page to be in the toctree so that it appears on the sidebar on the site. It can either be its own new page or it can be made into a section in the "Outputs of tedana" page, replacing the "Visual Reports" section. Either way, we'll need to also either remove the current "Visual Reports" section or make it very clear that those figures do not, on their own, reflect the standard outputs of tedana.

File: docs/reporting.rst
Comment (eurunuela) [reply docs/reporting.rst:65]: That's why I brought the OHBM demo up. We could add the link to cover for the interactiveness.

Comment (tsalo) [reply docs/reporting.rst:65]: Sorry, I mean that, in the actual report, these figures are not interactive. They're still static. I didn't keep up with the report PR as much as I should have, but I assume that brainsprite was either dropped or pushed off to a future pull request.

Comment (tsalo) [reply docs/reporting.rst:65]: But yes I agree that linking to example reports would be very useful.

Comment (eurunuela) [reply docs/reporting.rst:65]: Yes, the brainsprite stuff was pushed off to a future PR. Okay, if the other agree, I'll change my review to request changes.

**Comment by eurunuela** [comment]
> I have a couple of minor requested changes, but we also need for this new page to be in the toctree so that it appears on the sidebar on the site. It can either be its own new page or it can be made into a section in the "Outputs of tedana" page, replacing the "Visual Reports" section. Either way, we'll need to also either remove the current "Visual Reports" section or make it very clear that those figures do not, on their own, reflect the standard outputs of tedana.

I'm sorry. I've been so offline from tedana recently. I hope you like the changes I just pushed.

We can discuss them during out meeting tomorrow 😉

File: docs/reporting.rst
Comment (jbteves) [inline docs/reporting.rst:46]:
```suggestion
* **Rho Scree Plot:** This scree plot provides a view of the components ranked by `Rho`.
```

File: docs/reporting.rst
Comment (jbteves) [inline docs/reporting.rst:47]:
```suggestion
  As in the previous plot, each dot represents a component. The color of the dot informs us
```

File: docs/reporting.rst
Comment (jbteves) [reply docs/reporting.rst:68]: So should we just not list the plot as interactive?

File: requirements.txt
Comment (jbteves) [inline requirements.txt:10]: I believe that as of the latest commit, we're pulling dependencies from `tedana/info.py`

**Review by jbteves — CHANGES_REQUESTED** [review]
Only reviewed docs instead of code since I'm not familiar with the python-JS cross-talk employed with this PR.

File: requirements.txt
Comment (eurunuela) [reply requirements.txt]: My bad, I wasn't up to date with master. The latest commit should address this. Thank you for spotting that!

**Comment by eurunuela** [comment]
> Only reviewed docs instead of code since I'm not familiar with the python-JS cross-talk employed with this PR.

Thank you so much for the review @jbteves ! 

I'm not sure I understand what you mean by this though: _"So should we just not list the plot as interactive?"_

I added a link to the OHBM demo to the very top of the reporting page, so people can play around with it. The images in the docs are all static.

File: docs/reporting.rst
Comment (emdupre) [inline docs/reporting.rst:8]:
```suggestion
information on how to take advantage of the interactive functionalities.
```

File: docs/reporting.rst
Comment (emdupre) [inline docs/reporting.rst:19]: I don't think this reflects the state of the reports any more, unfortunately, since we removed the individual component view. The figure should be updated

File: docs/reporting.rst
Comment (emdupre) [inline docs/reporting.rst:41]: Should we explain what the axes are ?

File: docs/reporting.rst
Comment (emdupre) [inline docs/reporting.rst:68]:
```suggestion
component (selected in the summary view, see below).
```

File: docs/reporting.rst
Comment (emdupre) [inline docs/reporting.rst:86]:
```suggestion
```

File: docs/reporting.rst
Comment (emdupre) [reply docs/reporting.rst:68]: Instead here we should mention the PNG views and showcase one (or two, if you want to show idealized "accepted" and "rejected")

File: docs/reporting.rst
Comment (emdupre) [inline docs/reporting.rst:95]:
```suggestion
As previously mentioned, all summary plots in the report allow user interactions. The list of permitted
```

File: docs/reporting.rst
Comment (emdupre) [inline docs/reporting.rst:96]: In the current view, the toolbar is only on the Kappa-Rho plot, so we likely need to rephrase this section accordingly.

**Review by emdupre — CHANGES_REQUESTED** [review]
Thanks again, @eurunuela and @javiergcas ! I think this will be a huge help to users.

I've made a few comments  / suggestions to bring the docs in line with the current reporting functionality.

File: docs/reporting.rst
Comment (emdupre) [reply docs/reporting.rst:68]: +1 on this route -- I've made suggestions to that effect in my review !

File: docs/reporting.rst
Comment (eurunuela) [reply docs/reporting.rst]: [1b59407](https://github.com/ME-ICA/tedana/pull/465/commits/1b594074fdfa0fcc48f6cd28caf4fd3e397b13c3) should take care of this.

File: docs/reporting.rst
Comment (eurunuela) [reply docs/reporting.rst:19]: [1b59407](https://github.com/ME-ICA/tedana/pull/465/commits/1b594074fdfa0fcc48f6cd28caf4fd3e397b13c3) should take care of this.

I included the individual component view section in the end, as we do show a static view of individual components.

Comment (emdupre) [reply docs/reporting.rst:19]: That looks right, thanks !! Should we note that the individual component view will be empty by default, since no component is pre-selected ?

File: docs/reporting.rst
Comment (emdupre) [inline docs/reporting.rst:94]: This is what I was thinking of :smile_cat: Maybe let's move it up around L19 ?

**Review by emdupre — APPROVED** [review]
One [small suggestion](https://github.com/ME-ICA/tedana/pull/465/files#r507217416) from me, and I think this looks great. Thanks again !

File: docs/reporting.rst
Comment (jbteves) [inline docs/reporting.rst:31]:
```suggestion
  and size. In particular, color informs about its classification
```

File: docs/reporting.rst
Comment (jbteves) [inline docs/reporting.rst:39]:
```suggestion
* **Kappa Scree Plot:** This scree plot provides a view of the components ranked by `Kappa`.
```

File: docs/reporting.rst
Comment (jbteves) [inline docs/reporting.rst:40]:
```suggestion
  As in the previous plot, each dot represents a component. The color of the dot informs us
```

**Review by jbteves — CHANGES_REQUESTED** [review]
Found a few typos, but otherwise looks good with the continuous-docs render.

**Comment by eurunuela** [comment]
> Found a few typos, but otherwise looks good with the continuous-docs render.

Thank you! I thought I had already corrected those.

File: docs/reporting.rst
Comment (eurunuela) [reply docs/reporting.rst:94]: Yes! That's a great idea. Thank you!

**Review by jbteves — APPROVED** [review]
LGTM, thanks @eurunuela and @javiergcas !

**Review by emdupre — APPROVED** [review]
LGTM ! :rocket: Merging now. Thanks again !!

**Pull request merged**
